uint32 tow
float64 lat
float64 lon
float64 height
uint16 h_accuracy
uint16 v_accuracy
uint8 n_sats
uint8 flags
